technology-considerations/s3-user-access-to-objects.md

```diff
@@ -0,0 +1,121 @@
+# Restricting User Access To S3 Objects
+
+## Problem
+
+Ensure that TDR users are not able to overwrite/view another user’s objects in the TDR AWS S3 upload bucket.
+
+Potential options to resolve the problem are to use:
+* AWS Cognito;
+* Signed URLS.
+
+The purpose of the spike was to see whether using AWS Cognito was a feasible option.
+
+AWS Cognito has some advantages over using signed URLs:
+* Using signed URLs means an eavesdropper, potentially, can grab the signed url and use it until it expires
```

Comment:
We're discussing this on Slack, but just to note it here: AWS signed URLs use HTTPS, so an eavesdropper on a public network can't see the secret part which is in the request parameters or the path.

Reply:
Moved to the architectural decisions folder and updated to that format